src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,29 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import { Loader } from '../Loader';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
+import { getUser } from '../../api';
+
+type Props = {
+  modalInfo: Todo;
+  setModalInfo: (todo: Todo | null) => void;
+};
+
+export const TodoModal: React.FC<Props> = ({ modalInfo, setModalInfo }) => {
+  const [user, setUser] = useState<User | null>(null);
+  const [isLoading, setIsLoading] = useState(false);
+
+  useEffect(() => {
+    getUser(modalInfo.userId)
+      .then(setUser)
+      .finally(() => setIsLoading(true));
```

Comment:
You should do vice versa, set loader before starting loading, and then remove in finally